docs/guides/deploy-guide/examples/testbed.md

```diff
@@ -60,18 +65,18 @@ network_availability_zone = "nova"
 
 The OSISM Testbed requires at least the following project quota when using the default flavors:
 
-**Quantity** | **Resource**         | **Note**
--------------|----------------------|-------------------------
-4            | Instances            | 28 VCPUs + 104 GByte RAM
-9            | Volumes              | 90 GByte volume storage
-1            | Floating IP          |
-1            | Keypair              |
-3            | Security group       |
-16           | Security group rules |
-1            | Network              |
-1            | Subetwork            |
-6            | Ports                |
-1            | Router               |
+| **Quantity** | **Resource**         | **Note**                                      |
+|--------------|----------------------|-----------------------------------------------|
+| 4            | Instances            | 28 VCPUs + 112 GByte RAM (3 modes, 1 manager) |
+| 9            | Volumes              | 90 GByte volume storage                       |
+| 1            | Floating IP          |                                               |
```

Comment:
Does it makes sense to only list the instances & volumes here?

Reply:
No, i don't think so. The table just provides a overview about the needed resources. 
This is useful for general planning purposes. I think more detailed information is not needed here.